src/cart/src/cart/crt_init.c

```diff
@@ -82,12 +82,10 @@ mem_pin_workaround(void)
 		D_GOTO(exit, crt_rc = -DER_MISC);
 	}
 
-	/** Disable fastbins */
+	/* Disable fastbins; this option is not available on all systems */
 	rc = mallopt(M_MXFAST, 0);
-	if (rc != 1) {
+	if (rc != 1)
 		D_ERROR("Failed to disable malloc fastbins: %d\n", errno);
```

Comment:
This should be a warning or info then, but not an error.

Reply:
done